test/components/evaluators/test_faithfulness_evaluator.py

```diff
@@ -191,6 +193,49 @@ def test_run_missing_parameters(self, monkeypatch):
         with pytest.raises(TypeError, match="missing 3 required positional arguments"):
             component.run()
 
+    def test_run_handles_nan(self, monkeypatch):
```

Comment:
For the same reason as above I suggest `test_run_returns_nan_on_failure` instead.

Reply:
or `test_run_returns_nan_raise_on_failure_false`